engine/src/deploy_sign.cpp

```diff
@@ -1034,7 +1034,8 @@ static bool MCDeploySignWindowsAddTimeStamp(const MCDeploySignParameters& p_para
 	// the sig.
 	if (t_request != nil)
 	{
-		t_request -> blob -> signature = nil;
+        if (t_request -> blob != nil)
```

Comment:
That's not quite right. If t_request is not nil it needs to be freed. If it has a non-nil blob field it needs its signature unhooked as that's borrowed from elsewhere.